src/core/server/http/http_server.ts

```diff
@@ -111,6 +112,7 @@ export class HttpServer {
         getAuthHeaders: this.authRequestHeaders.get,
       },
       isTlsEnabled: config.ssl.enabled,
+      csp: config.csp,
```

Comment:
`yarn kbn bootstrap` fails because `createPluginSetupContext` doesn't contain `csp` field

Reply:
I think this should be fixed now.